utils/file_checking.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+
+
+def check_number_of_files(directory: pathlib.Path, n_files: int) -> bool:
```

Comment:
This seems like a great place to start with the file preprocessing as we discussed!